.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -61,8 +80,84 @@ dynamodb_BatchGetItem:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-example-table-read-write-batch.html#dynamodb-example-table-read-write-batch-reading
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createclientv3
+            - description: Get the items.
+              snippet_tags:
+                - dynamodb.JavaScript.batch.GetItemV3
+        - sdk_version: 2
+          github: javascript/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v2/developer-guide/dynamodb-example-table-read-write-batch.html#dynamodb-example-table-read-write-batch-reading
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.JavaScript.batch.GetItem
   services:
     dynamodb: {BatchGetItem}
+dynamodb_DescribeTable:
+  title: Get information about a &DDB; table
+  title_abbrev: Get information about a table
+  synopsis: get information about a &DDB; table.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-examples-using-tables.html#dynamodb-examples-using-tables-describing-a-table
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createclientv3
+            - description: Describe the table.
+              snippet_tags:
+                - dynamodb.JavaScript.table.describeTableV3
+        - sdk_version: 2
+          github: javascript/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v2/developer-guide/dynamodb-examples-using-tables.html#dynamodb-examples-using-tables-describing-a-table
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.JavaScript.table.describeTable
+  services:
+    dynamodb: {Get}
+dynamodb_Get:
+  title: Get an item from a &DDB; table using the &DDB; document client using an &AWS; SDK
+  title_abbrev: Get an item from a table using the document client
+  synopsis: get an item from a &DDB; table using the &DDB; document client.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-example-dynamodb-utilities.html#dynamodb-example-document-client-get
+          excerpts:
+            - description: Create the &DDB; document client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createdocclientv3
+            - description: Create the &DDB; client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createdocclientv3

```

Comment:
Wrong snippet tag: this snippet is the doc client snippet.
Also, I'd move the "Create the DDB client" excerpt above the document client excerpt, because the code reads better that way.

Reply:
fixed